src/securityPolicy.c

```diff
@@ -154,10 +161,15 @@ security_policy_t policyForSignTxInput()
 
 // For each transaction (third-party) address output
 security_policy_t policyForSignTxOutputAddress(
+        bool includePoolRegistrationCertificate,
```

Comment:
this is bad naming. I think we should use something more like "isSigningPoolRegistrationAsOwner"